webapp/packages/plugin-connections/src/ConnectionForm/Options/getDefaultConfigurationType.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import type { DBDriver } from '@cloudbeaver/core-connections';
+import { DriverConfigurationType } from '@cloudbeaver/core-sdk';
+
+export function getDefaultConfigurationType(driver: DBDriver) {
+  const isCustomConfiguration = driver?.configurationTypes.includes(DriverConfigurationType.Custom) && driver.mainProperties.length > 0;
```

Comment:
`&& driver.mainProperties.length > 0` is extra and can be omitted
for example the first part `driver?.configurationTypes.includes(DriverConfigurationType.Custom)` will be true only when `driver.mainProperties.length > 0` otherwise it will be false